### DISCUSSION TITLE
Resetting notification to default ?

### DISCUSSION THREAD

**Discussion by nyue** [python-gitlab/python-gitlab#2095] [Q&A]
I see options and example code to customize notification here

https://python-gitlab.readthedocs.io/en/stable/gl_objects/notifications.html

Is there a way or a call to reset the notification to factory default ?

**Comment by nejch** [comment]
This question would be more for upstream gitlab, but you can maybe try - the default level is Participating. I'm assuming you mean global settings.
```
settings = gl.notificationsettings.get()
settings.level = gitlab.NOTIFICATION_LEVEL_PARTICIPATING
settings.save()
```
Not sure if this overrides custom settings. Otherwise you'd need to loop and set them all to True/False as needed. And also loop over groups/projects you're a member of for non-global settings probably.

**Comment by nejch** [comment]
@nyue I think this was answered but I'll move it to discussions in case anyone wants to add their own ideas :)
